packages/block-library/src/navigation-menu/edit.js

```diff
@@ -148,15 +165,17 @@ function NavigationMenu( {
 			</InspectorControls>
 
 			<div className={ navigationMenuClasses } style={ navigationMenuInlineStyles }>
-				{ isRequesting && <><Spinner /> { __( 'Loading Navigation…' ) } </> }
-				{ pages &&
+				<div ref={ scrollContainerRef } className="wp-block-navigation-menu__scroll-container">
```

Comment:
The scroll container is unnecessary; the styles can just as easily be applied to `wp-block-navigation-menu`.
Also, it seems to be causing small navs to get a horizontal scrollbar:

<img width="685" alt="Screen Shot 2019-11-13 at 4 28 26 pm" src="https://user-images.githubusercontent.com/8096000/68736558-0d09c980-0635-11ea-8411-c4e86f450097.png">